### PR TITLE
Add a footer

### LINKED ISSUE #3: Footer
Add a footer in welcome page.

### PR DESCRIPTION
**Pull request by zunyangzy03**
I have added a grey color footer in the welcome page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 <body>
     <h1 style="color:red">This is my welcome page</h1>
+
+    <footer style="color:grey">footer by zunyang</footer>
```
Comment (HermanLKH) [inline index.html]: Add your student id in the footer and make it bold.

File: index.html [hunk 2]
```diff
@@ -9,6 +9,8 @@
 
 <body>
     <h1 style="color:red">This is my welcome page</h1>
```
Comment (HermanLKH) [inline index.html:11]: Make the header bold and add a '!' after 'This is my welcome page'.

**Review by HermanLKH — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (zunyangzy03) [reply index.html]: Sure, can do.

File: index.html (hunk 2 above)
Comment (zunyangzy03) [reply index.html:11]: Ok, will do.

**Review by HermanLKH — APPROVED** [review]
Goodjob! @zunyangzy03

**Pull request merged**